EventGenerator/src/GenEventCounter_module.cc

```diff
@@ -27,17 +27,24 @@ namespace mu2e {
   class GenEventCounter : public art::EDProducer {
     GenEventCount::count_t seenEvents_;
   public:
-    struct Config {};
+    struct Config {
+      using Name=fhicl::Name;
+      using Comment=fhicl::Comment;
+      fhicl::Atom<bool> allowOtherGenCounters{Name("allowOtherGenCounters"), Comment("Allow other gen counters in a subrun"), false};      
+    };
```

Comment:
For this, is there an easy way to remove a module from the trigger path or set the label to a trivial producer? Otherwise, this generation module would need a parallel set of paths defined for any generation, doubling maintenance of paths. I could also add a flag like `doNotCount_` that tells the GenEventReader to not count or produce an output?

I tried simply dropping the count produced by this module on output, but it throws an exception when it sees an event count already exists when attempting to produce its count.

Reply:
I've reverted to the previous version of the gen count producer.